UIKitMovie/UIKitMovie/Helpers/AsselderModelBuilder.swift

```diff
@@ -0,0 +1,25 @@
+// AsselderModelBuilder.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Сборщик экранов
+final class AsselderModelBuilder: AsselderBuilderProtocol {
+    // MARK: - Public Methods
+
+    func makeMovieModule(router: RouterProtocol) -> UIViewController {
```

Comment:
```suggestion
    func makeMovieModule(router: RouterProtocol) -> UIViewController {
```
назови экраны как-то по другому, MovieModule и FilmModule для меня выглядят одинаково